v1_push_api.json

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
Nope, I was just using it as a reference and you never gave your opinion on this:
>
Also, should the original v1 schema be included anywhere? Right now it is just at the top level.
>